lighthouse-core/audits/time-to-interactive.js

```diff
@@ -78,17 +78,21 @@ class TTIMetric extends Audit {
       const fMPts = timings.fMPfull + timings.navStart;
 
       // look at speedline results for 85% starting at FMP
-      let eightyFivePctVC = artifacts.Speedline.frames.find(frame => {
-        return frame.getTimeStamp() >= fMPts && frame.getProgress() >= 85;
-      });
+      let eightyFivePctVC = artifacts.Speedline.frames === undefined ?
+        undefined :
+        artifacts.Speedline.frames.find(frame => {
+          return frame.getTimeStamp() >= fMPts && frame.getProgress() >= 85;
+        });
 
       // Check to avoid closure compiler null dereferencing errors
       if (eightyFivePctVC === undefined) {
         eightyFivePctVC = 0;
       }
 
       // TODO CHECK these units are the same
-      const visuallyReadyTiming = (eightyFivePctVC.getTimeStamp() - timings.navStart) || 0;
+      const visuallyReadyTiming = eightyFivePctVC === 0 ?
```

Comment:
the only time `eightyFivePctVC === 0` is if `eightyFivePctVC === undefined` above, so just move `visuallyReadyTiming = 0` into there?